sys/drv/pit.c

```diff
@@ -12,48 +12,78 @@
 #define PIIX4_IDE_VENDOR_ID 0x8086
 #define PIIX4_IDE_DEVICE_ID 0x7111
 
+typedef union {
+  /* assumes little endian order */
+  struct {
+    uint32_t lo;
+    uint32_t hi;
+  };
+  uint64_t val;
+} counter_t;
```

Comment:
It looks like some universal structure.
Could you add some explanation why we need that structure and why it's specific for PIT timer?

Reply:
Good idea!